src/acquisition/covid_hosp/common/database.py

```diff
@@ -12,17 +11,15 @@
 # first party
 import delphi.operations.secrets as secrets
 from delphi.epidata.common.logger import get_structured_logger
-
-Columndef = namedtuple("Columndef", "csv_name sql_name dtype")
+from delphi.epidata.common.covid_hosp.covid_hosp_schema_io import CovidHospSomething
 
 class Database:
 
+  DATASET_NAME = None
+
   def __init__(self,
                connection,
-               table_name=None,
-               hhs_dataset_id=None,
-               columns_and_types=None,
-               key_columns=None):
+               chs = CovidHospSomething()):
```

Comment:
is this `chs` argument here just to do dependency injection in tests?  can we remove it from the function prototype and use `unittest.mock.patch` instead?

Reply:
Yep! I also removed some of the now-missing constructor arguments from the docstring.